gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/metadata/MetaData.java

```diff
@@ -37,6 +52,68 @@
   }
 
   /**
+   * Reads the meta data from a specified file. Each line is spit into a (label, metadata) tuple
+   * and put into a dataset. The latter can be used to broadcast the metadata to the mappers.
+   *
+   * @param path path to metadata csv file
+   * @param config gradoop configuration
+   * @return (label, metadata) tuple dataset
+   */
+  public static DataSet<Tuple2<String, String>> fromFile(String path, GradoopFlinkConfig config) {
+    return config.getExecutionEnvironment()
+      .readTextFile(path)
+      .map(line -> {
+          String[] tokens = line.split(CSVConstants.TOKEN_DELIMITER, 2);
+          return Tuple2.of(tokens[0], tokens[1]);
+        })
+      .returns(new TypeHint<Tuple2<String, String>>() { });
+  }
+
+  /**
+   * Reads the meta data from a specified csv file. The file can be either located in a local file
+   * system or in HDFS.
+   *
+   * @param path path to metadata csv file
+   * @param hdfsConfig file system configuration
+   * @return meta data
+   * @throws IOException
+   */
+  public static MetaData fromFile(String path, Configuration hdfsConfig) throws IOException {
+    FileSystem fs = FileSystem.get(hdfsConfig);
+    Path file = new Path(path);
+    Charset charset = Charset.forName("UTF-8");
+
+    try (BufferedReader br = new BufferedReader(new InputStreamReader(fs.open(file), charset))) {
+      return MetaDataParser.create(br.lines()
+        .map(line -> line.split(CSVConstants.TOKEN_DELIMITER, 2))
+        .map(tokens -> Tuple2.of(tokens[0], tokens[1]))
+        .collect(Collectors.toList()));
+    }
+  }
+
+  /**
+   * Returns the vertex labels available in the meta data.
+   *
+   * @return vertex labels
+   */
+  public Set<String> getVertexLabels() {
+    return metaData.keySet().stream()
+      .filter(l -> Character.isUpperCase(l.charAt(0)))
```

Comment:
Do we specify somewhere that vertices have to start with an upper case letter and edges with a lower case one?